ingress/src/test/java/com/redhat/service/bridge/ingress/api/IngressAPITest.java

```diff
@@ -28,6 +32,13 @@
 @QuarkusTest
 public class IngressAPITest {
 
+    private static final String TOPIC_NAME = "topicName";
```

Comment:
As far as I see this is intended to be the bridge name/id, can you please rename it accordingly?